syntax = "proto3";

package llmservice;

service LLMService {
  rpc GetLLMAnswer(LLMQuery) returns (LLMAnswerResponse);
}

message LLMQuery {
  string request_id = 1;
  string query = 2;
  string context = 3;
}

message LLMAnswerResponse {
  string request_id = 1;
  string answer = 2;
}
